website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,239 @@
+
+We have integration options with enterprise identity providers like Okta, Microsoft Entra ID, and Keycloak. Connect these identity providers through either OpenID Connect or SAML 2.0 protocol. For a step-by-step guide, read our [how-to guide for single sign-on](/how-to/sso).
+
+Another major benefit to using SSO is that when you connect Unleash to your identity provider, you will be able to synchronize your user groups out of the box. That way, your group configuration reflects organizational-level changes automatically. Follow our step-by-step guide to set up [user group syncing](/how-to/how-to-set-up-group-sso-sync). With SSO integration, groups defined in your identity provider are directly mapped to Unleash access groups. This means:
+
+-   New team members are automatically assigned correct permissions.
+-   Organizational restructures are reflected immediately.
+-   Consistent access controls are enabled across all enterprise systems.
+
+By integrating Unleash with these systems, your organization can prove that every engineer accessing feature flags undergoes a centralized, unified authentication process.
+
+How do you translate this into something verifiable and auditable for security reviews? What’s great is that every authentication event in Unleash is logged with detailed metadata including timestamp, IP address, user agent, and authentication method used, providing audit trails when you undergo security reviews. This shows you have a transparent, auditable system where every access can be traced, logged, and validated. So not only are you meeting compliance standards with authentication controls in place, but Unleash also automatically maintains a record for you that proves it.
+
+### Use SCIM to automate user management at scale
+
+Let’s consider another scenario. You are not setting up feature flag access for 10 users, but 100 or 1000 users. When these users are spread across various teams with access to different projects at various permission levels, how do you manage this effectively while maintaining a security-first approach?
+
+User management at scale is difficult without automation. It’s also costly to manually track and change all user permissions over time.
+
+To solve this, Unleash uses [SCIM protocols (System for Cross-domain Identity Management)](https://scim.cloud/) to help you automatically provision and de-provision user accounts. When an employee joins or leaves your organization, their feature flag access can be automatically adjusted without manual changes. You won’t have to consider all the security implications of user accounts as your teams evolve.
+
+By enabling SCIM in Unleash, you can:
+
+-   [Provision and de-provision users](/reference/scim) (team members) as they are joining or leaving your organization.
+-   Automatically create and delete user groups.
+-   Sync group membership in real-time.
```

Comment:
SCIM isn't real time, it's up to the provider but generally around a 40 minute lag

```suggestion
-   Sync group membership.
```